lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -789,7 +788,15 @@ def quickchip_payment?(payment)
 
       def card_from_response(response)
         # StripePI puts the AVS and CVC check significantly deeper into the response object
-        response['card'] || response['active_card'] || response['source'] || response.dig('charges', 'data', 0, 'payment_method_details', 'card', 'checks') || {}
+        response['card'] || response['active_card'] || response['source'] ||
+          response.dig('charges', 'data', 0, 'payment_method_details', 'card', 'checks') ||
+          response.dig('latest_attempt', 'payment_method_details', 'card', 'checks') || {}
+      end
+
+      def avs_code_from(card_checks)
+        return 'I' if card_checks['address_postal_code_check'].nil? && card_checks['address_line1_check'].nil? && card_checks['address_zip_check'].nil?
```

Comment:
Would 'I' be considered the a default value for Stripe or card brands?

Reply:
It looks like 'I' is what is returned if the address line1 or zip are both unchecked. So I guess we can assume `nil` means unchecked or they are just not returning the results of the check.

Reply:
That's a good question. the "I", just means that it wasn't checked, but I'm not sure who defined that code. I was just talking to @rachelkirk about this and I think I am going to remove these nil checks and only set a code if stripe is specifically sending us a code.